src/components/AdminPage/UserForm/Users.js

```diff
@@ -1,28 +1,26 @@
 import { Row, Col, Button, Container } from "react-bootstrap";
-import styles from "./NewUser.module.css";
+import styles from "./Users.module.css";
 import removeUser from "../../../RestaurantApi/Users/RemoveUser";
 import getAllUsers from "../../../RestaurantApi/Users/GetAllUsers";
 import { useEffect, useState } from "react";
 
-const NewUser = ({ users }) => {
+const Users = () => {
   const [allUsers, setAllUsers] = useState([]);
   const [isReloaded, setIsReloaded] = useState({});
   const removingUser = (id) => {
     removeUser(id).then(() => setIsReloaded({}));
   };
 
-  useEffect(() => {
-    const getUsers = async () => {
-      const savedUsers = await getAllUsers();
+  const getUsers = async () => {
+    const savedUsers = await getAllUsers();
+    try {
       setAllUsers(savedUsers);
```

Comment:
you have to put await getAllUsers() into try catch block because now it is not catched adn throwing an error. For now you can just console.log it but have in mind that it should be show to the client that something went wrong on the server (some info in place were users should be)